lib/jekyll-import/importers/drupal_common.rb

```diff
@@ -175,12 +178,9 @@ def post_data(_sql_post_data)
 
       def validate(options)
         %w(dbname user).each do |option|
-          if options[option].nil?
-            abort "Missing mandatory option --#{option}."
-          end
+          abort "Missing mandatory option --#{option}." if options[option].nil?
```

Comment:
If the option is a true/false value, then e.g. `--skip-posts=false` will cause `options[:skip_posts]` to be falsey. This would be incorrect behavior, since the option _was_ given, it was just given a false value.

I think doing `unless options.key?(option)` is probably best.